lib/streamlit/util.py

```diff
@@ -403,22 +404,24 @@ def is_function(x):
 
 
 def get_streamlit_file_path(*filepath):
-    """Return the full path to a filepath in ~/.streamlit.
+    """Return the full path to a file in ~/.streamlit.
 
-    Creates ~/.streamlit if needed.
```

Comment:
Since this PR removes this (which I think is the right thing to do), please try deleting config.toml and credentials.toml and then running Streamlit to see if anything breaks.

The missing credentials.toml should cause Streamlit to ask for the user's email, and then create the credentials file.

The missing config.toml should not make any difference. Streamlist should just work.